### DISCUSSION TITLE
Can i join team ?

### DISCUSSION THREAD

**Discussion by Nishant9947** [ROM-builders/temporary#13480] [General]
Device Tree (-b EXS) :- https://github.com/Ultron12bot/android_device_asus_X01AD/tree/EXS

Tg :- https://t.me/Nishant9947

Currently maintaining Evolution X for Asus_X01AD from many months.

https://cirrus-ci.com/build/6553741103988736

Not a fake user or untrusted:-
https://cirrus-ci.com/build/6579762305957888

**Comment by Nishant9947** [comment]
@Apon77

**Comment by Apon77** [comment]
mismatch github and telegram name cant be added for safety

**Reply by Nishant9947** [reply]
Oh bruh , the name is taken 
& Ultron12bot was my first username with which i created account github.

But now I changed on both git n tg.
Should i tag u in rom builder.

https://cirrus-ci.com/build/6579762305957888
